specs/latest/2.0/index.html

```diff
@@ -734,18 +734,11 @@ <h2 class="no-toc">Table of contents</h2>
   void texStorage3D(GLenum target, GLsizei levels, GLenum internalformat, GLsizei width,
                     GLsizei height, GLsizei depth);
 
-  // WebGL1 legacy entrypoints:
+  // WebGL1 legacy DOM entrypoints:
   void texImage2D(GLenum target, GLint level, GLint internalformat,
-                  GLsizei width, GLsizei height, GLint border, GLenum format,
-                  GLenum type, ArrayBufferView? pixels);
-  void texImage2D(GLenum target, GLint level, GLint internalformat,
-                  GLenum format, GLenum type, TexImageSource? source); // May throw DOMException
-
-  void texSubImage2D(GLenum target, GLint level, GLint xoffset, GLint yoffset,
-                     GLsizei width, GLsizei height,
-                     GLenum format, GLenum type, ArrayBufferView? pixels);
+                  GLenum format, GLenum type, TexImageSource source); // May throw DOMException
```

Comment:
Maybe I'm confused about which direction you were going with this pull request, but to match the nullability of the WebGL 1.0 entry point this should be "TexImageSource?".